test/submission.e2e.js

```diff
@@ -22,6 +22,22 @@ const manuscript = {
 }
 
 const getPageUrl = ClientFunction(() => window.location.href)
+const autoRetry = async (fn, timeout = 5000) => {
+  const delay = 100
+  const start = Date.now()
+  while (true) {
```

Comment:
Should we have a timeout here ... even if its 10 mins to prevent this hanging adinfinitum?

Reply:
The function either returns on line 31 or throws on line 34. I think the only way it could hang is if `fn` doesn't return but that seems quite unlikely.